src/entities/developer/model/types.ts

```diff
@@ -0,0 +1,8 @@
+export type TypeDeveloper = {
+  id: number;
+  name: string;
+  role: string;
+  avatarSrc: string;
+  githubName: string;
+  githubLink: string;
+};
```

Comment:
if you wanted to type your json, you can just infer the `results` type with `ReturnType<typeof obj>`